test/smoke/appServers/build.gradle

```diff
@@ -56,6 +56,7 @@ subprojects {
 		doLast {
 			def jresFile = file("$resourcesOutputDir/${appServerShortName}.jre.txt")
 			jresFile.getParentFile().mkdirs()
+			jresFile.delete()
```

Comment:
Is this necessary or should it be part of the `clean` task?

Reply:
i think i tried in clean, but `$resourcesOutputDir/${appServerShortName}` did not resolve, i think clean was too early, probably there's a way, but my gradle isn't good enough yet

Reply:
I think clean might already be deleting these files...I'll have to check.